EIPS/eip-2015.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 2015
+title: Wallet Update Chain JSON-RPC Method (`wallet_updateChain`)
+author: Pedro Gomes (@pedrouid)
+discussions-to: https://ethereum-magicians.org/t/add-wallet-methods-to-improve-dapp-to-wallet-interaction/1848/31
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-05-12
+requires: 155
+---
+
+## Simple Summary
+Wallets can update the active chain when connected to a Dapp but not vice-versa, with wallet_updateChain the Dapp will be able to request this change from the Wallet.
+
+## Abstract
+Dapp can request the Wallet to switch chains by providing the minimal parameters of chainId, networkId, rpcUrl and nativeCurrency. The Wallet will display a UI element to inform the user of this change.
```

Comment:
Can you quote the field names or change them to regular text?

Reply:
Also they are inconsistent with the specification, for example `networkId` vs. `networkID`, but `chainId` has a lowercase `d` letter in both cases.